src/compute/src/metrics.rs

```diff
@@ -69,6 +82,12 @@ impl ComputeMetrics {
                 help: "Whether maintenance is currently occuring.",
                 var_labels: ["worker_id"],
             )),
+            timely_step_duration_seconds: registry.register(metric!(
+                name: "mz_timely_step_duration_seconds",
+                help: "The time spent in each compute step_or_park call",
+                const_labels: {"cluster" => "compute"},
+                buckets: mz_ore::stats::histogram_seconds_buckets(0.000_128, 32.0),
```

Comment:
I was about to suggest having this metric also for the storage cluster, but based on the name and the label you already thought of that. Did you plan to make that a separate PR?

Reply:
I hadn't looked at it yet, tbh. Wanted to make sure this one made the cutoff for next week's release candidate, so quickly typed it up and that's how far I've gotten. Follow up PR makes sense to me